packages/bpk-component-phone-input/src/BpkPhoneInput.js

```diff
@@ -211,6 +210,8 @@ BpkPhoneInput.propTypes = {
     id: PropTypes.string,
     name: PropTypes.string,
     label: PropTypes.string,
+    className: PropTypes.string,
+    wrapperClassName: PropTypes.string,
```

Comment:
Great catch!

Reply:
All credit goes to eslint, I just fixed the issue that was raised (so the update is picking new issues 👏 )